geocoder_tester/world/france/iledefrance/test_addresses.csv

```diff
@@ -899,9 +896,9 @@ query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_
 9 Place de la Hayette Jouy-le-Moutier,,,,9 Place de la Hayette,,,,95280
 9 Place de la Hayette 95280,,,,9 Place de la Hayette,,,Jouy-le-Moutier,
 9 Place de la Hayette,2.038193,49.007774,,9 Place de la Hayette,,,,95280
-8 Allée du Pré Aux Cerfs Luzarches,,,,8 Allée du Pré Aux Cerfs,,,,95270
-8 Allée du Pré Aux Cerfs 95270,,,,8 Allée du Pré Aux Cerfs,,,Luzarches,
-8 Allée du Pré Aux Cerfs,2.419913,49.119635,,8 Allée du Pré Aux Cerfs,,,,95270
+8 Allée du Pré Aux Cerfs Luzarches,,,,8 Allée du Pré aux Cerfs,,,,95270
+8 Allée du Pré Aux Cerfs 95270,,,,8 Allée du Pré aux Cerfs,,,Luzarches,
+8 Allée du Pré Aux Cerfs,2.419913,49.119635,,8 Allée du Pré aux Cerfs,,,,95270

```

Comment:
Note: you can use the `--loose-compare` option not to check diacritic signs or case.